editor.js

```diff
@@ -147,7 +147,16 @@ function extension_editor_finish_run() {
 	$("#xkit-editor-new").click(function() {
 
 		XKit.window.show("Create extension","<input type=\"text\" id=\"xkit-editor-filename\" placeholder=\"Filename (eg: my_extension)\"><br/>No spaces or special characters.","question","<div id=\"xkit-editor-create-extension\" class=\"xkit-button default\">OK</div><div id=\"xkit-close-message\" class=\"xkit-button\">Cancel</div>");
-
+		
+		$("#xkit-editor-filename").focus();
+		
+		$("#xkit-editor-filename").on('keydown', function(event) {
+			if(event.which == 13 || event.keyCode == 13) {
+				console.log("Pressed enter");
```

Comment:
Oh shit, yeah. Sorry forgot to remove that after having to test around, fixed now